apps/nowcasting-app/components/hooks/use-time-now.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from "react";
+import useGlobalState, { get30MinNow, getFloor30MinNow } from "../globalState";
```

Comment:
remove getFloor30MinNow it's not used anymore here

Reply:
Thanks